src/store.ts

```diff
@@ -75,6 +77,18 @@ const store: TypedVuexStore = new TypedVuexStore({
     ) {
       state.toggleableRequirementChoices = toggleableRequirementChoices;
     },
+    setSelectableRequirementChoices(
+      state: VuexStoreState,
+      selectableRequirementChoices: AppSelectableRequirementChoices
+    ) {
+      state.selectableRequirementChoices = selectableRequirementChoices;
+    },
+    setUserRequirementChoices(
+      state: VuexStoreState,
+      selectableRequirementChoices: AppSelectableRequirementChoices
+    ) {
+      state.selectableRequirementChoices = selectableRequirementChoices;
+    },
```

Comment:
What is the purpose of `setUserRequirementChoices(...)`? I don't think this is necessary since we have `setSelectableRequirementChoices(...)`. @SamChou19815 can you also confirm?

Reply:
ah yes, this seems to be duplicated

Reply:
lol whoops that was back before we named it selectable requirements and I didn't catch it. Thanks